lib/Migrator/FilesMigrator.php

```diff
@@ -86,6 +87,46 @@ public function __construct(
 		$this->l10n = $l10n;
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getEstimatedExportSize(IUser $user): int {
+		$uid = $user->getUID();
+
+		$userFolder = $this->root->getUserFolder($uid);
+
+		$size = $userFolder->getSize() / 1024;
+
+		// Export file itself is not exported so we subtract it if existing
+		try {
+			$exportFile = $userFolder->get(ExportDestination::EXPORT_FILENAME);
+			if (!($exportFile instanceof File)) {
+				throw new \InvalidArgumentException('User export is not a file');
+			}
+
+			$size -= $exportFile->getSize() / 1024;
+		} catch (NotFoundException $e) {
+			// No size subtraction needed if export file doesn't exist
+		}
+
+		try {
+			$versionsFolder = $this->root->get('/'.$uid.'/'.FilesVersionsStorage::VERSIONS_ROOT);
```

Comment:
Does this crash if files_versions app is disabled?

Reply:
Doesn't crash as it is caught and ignored